daemon/src/cfd_actors.rs

```diff
@@ -116,9 +116,17 @@ pub async fn handle_oracle_attestation(
 
     for id in db::load_all_cfd_ids(&mut conn).await? {
         let cfd = load_cfd(id, &mut conn).await?;
-        let event = try_continue!(cfd
-            .decrypt_cet(&attestation)
-            .context("Failed to decrypt CET using attestation"));
+
+        let event = match cfd.decrypt_cet(&attestation) {
+            Ok(event) => event,
+            Err(e) => match e {
+                DecryptCetError::NoDlc(_) | DecryptCetError::CfdIsFinal => {
+                    tracing::trace!("Decrypting cet is a no-op: {:#}", e);
+                    None
+                }
+                _ => bail!(e),
```

Comment:
I don't think we should bail here. Just because one CFD fails to decrypt the CET doesn't mean we shouldn't handle the others.

Reply:
I think the bails here are proper errors. Something must of gone really wrong if this is happening?